moncli/entities/item.py

```diff
@@ -726,21 +733,43 @@ def change_column_value(self, column_value = None, get_column_values: bool = Non
                 updates : `moncli.entities.update.Update`
                     The item's updates.
         """
+        if id and title : 
+            raise ItemError(
+                'change_column_value_too_many_parameters',
+                self.id,
+                'Cannot use both "id" and "title" parameters.'
+            )
+        if not (id or title):
+            if isinstance(column_value, ColumnValue):
+                pass
+            else:
+                raise ItemError(
+                    'invalid_column_value',
+                    self.id,
+                    'Column value must be a properly formatted dict or str when using "id" or "title" parameters.'
+                )
+        if id or title:
+            value = column_value
+            if isinstance(column_value,(dict,str)):
+                if title:
+                    column_id= self.column_values[title].id
+                if id:
+                    column_id= id
+            else: 
+                raise ItemError(
+                    'invalid_column_value_entity',
+                    self.id,
+                    'Column Value must be a valid entities.column_value.ColumnValue instance when not using "id" or "title" parameters.'
+                )
 
         if get_column_values:
             args = list(args)
-            for arg in ['column_values.{}'.format(arg) for arg in api.DEFAULT_COLUMN_VALUE_QUERY_FIELDS]:
-                if arg not in args:
-                    args.append(arg)
-            args.extend(['id', 'name'])
 
         if column_value is None:
```

Comment:
I will update the documentation with this, but when column_value is null, we can assume that we are using an _id_ or _title_. In this case let's look up the column value via self.column_values and then set value to column_value.null_value.